src/components/Checkout/SocialShare.vue

```diff
@@ -88,20 +97,29 @@ export default {
 	},
 	data() {
 		return {
-			selectedLoanIndex: 0,
+			copyStatus: {
+				class: '',
+				disabled: false,
+				text: 'Copy Link'
+			},
+			maxMessageLength: 280,
 			message: '',
-			maxMessageLength: 280
+			selectedLoanIndex: 0,
 		};
 	},
 	computed: {
+		selectedLoan() {
+			return this.loans[this.selectedLoanIndex];
+		},
+		selectedLoanUrl() {
+			return `https://www.kiva.org/lend/${this.selectedLoan.id}`;
+		},
 		placeholderMessage() {
-			return `Why did you lend to ${this.loans[this.selectedLoanIndex].name}?`;
+			return `Why did you lend to ${this.selectedLoan.name}?`;
 		},
 		suggestedMessage() {
-			const selectedLoan = this.loans[this.selectedLoanIndex];
-			const { name } = selectedLoan;
-			const location = selectedLoan.geocode.city || selectedLoan.geocode.country.name;
-			return `Kiva is an easy way to make a real difference in someone's life. Will you join me in helping ${name} in ${location} to pursue their dream?`; // eslint-disable-line max-len
+			const location = this.selectedLoan.geocode.city || this.selectedLoan.geocode.country.name;
```

Comment:
Since the selected loan won't always be defined, I would recommend using lodash.get here:
```
import _get from 'lodash/get';
...
const location = _get(this, 'selectedLoan.geocode.city') || _get(this, 'selectedLoan.geocode.country.name');
```